### DISCUSSION TITLE
Making Scratch behavior consistent when clicking information boxes and block arguments

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7303] [Ideas]
When you click in a block argument, the whole content is selected, so that you can just write the new value, without having to select the old value before. Instead, when you click on a box in the sprite information area, the cursor shows up in the point you clicked, but the box value is not selected, so you have to delete every char/digit (or select the whole content yourself and delete it) before entering a new value. There is no reason why information boxes and block arguments should behave in a different way. And the way block arguments work is the most convenient.

**Comment by Waakul** [comment]
This is not the place to ask scratch to make changes. We're not the ST. And you didn't mention you want this as a new addon. And indeed, this is supposed to be an issue. (Not being rude or mean)

**Reply by mxmou** [reply]
This is the SA repo, so there's no reason to specify that this should be a SA feature and not a change in Scratch. Feature suggestions can be posted as issues or as discussions.

**Reply by Waakul** [reply]
> This is the SA repo, so there's no reason to specify that this should be a SA feature and not a change in Scratch. Feature suggestions can be posted as issues or as discussions.

It is literally spoken in a language that feels like it's asking us to fix a scratch issue, not adding it as an addon...

**Reply by BroJac5246** [reply]
> It is literally spoken in a language that feels like it's asking us to fix a scratch issue, not adding it as an addon...

They are asking us to fix a Scratch issue. But since this is the Scratch Addons repo, it's understood that it's a Scratch Addons feature request. Only if this were unfeasible or not useful would it be rejected.
